conda/recipes/cuxfilter/meta.yaml

```diff
@@ -15,19 +15,20 @@ source:
 
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
-  string: py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
 
 requirements:
   host:
     - python
     - setuptools
+    - cuda-version ={{ cuda_version }}
   run:
     - bokeh >=2.4.2,<=2.5
-    - cudf ={{ minor_version }}
+    - cudf ={{ version }}
     - cupy >=12.0.0
-    - cuspatial ={{ minor_version }}
-    - dask-cuda ={{ minor_version }}
-    - dask-cudf ={{ minor_version }}
+    - cuspatial ={{ version }}
+    - dask-cuda ={{ version }}
+    - dask-cudf ={{ version }}
```

Comment:
Wouldn't we want to keep `minor_version` here to allow for patch releases of these dependencies to be picked up?

Reply:
I picked it up from how dask-cudf had the recipe, but I get your point, if that's to accomadate for patch releases, then I agree, it's needed. I'll revert the changes. Thanks!

Reply:
fixed it!